javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,38 @@
+const width = 4;
+const height = 6;
+const area = width * height;
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+const bill = 20;
+const payment = 20;
+const change = bill - payment;
+console.log('value of change:', change);
+console.log('typeof change:', typeof change);
+
+const quizzes = 100;
+const midterm = 95;
+const final = 98;
+const grade = (quizzes + midterm + final) / 3;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+const firstName = 'Bryan';
+const lastName = 'Amora';
+let fullName = firstName + ' ' + lastName;
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+
+const pH = 9;
+const isAcidic = pH < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+const headCount = 250;
+const isSparta = headCount === 300;
+console.log('value of isSparta', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+const motto = (fullName += ' is the GOAT');
```

Comment:
Instead of updating the value of `fullName`, you should be updating the value of `motto`. So you should be doing the following

1. Assign the value of `fullName` to `motto`
2. Using the `+=` operator, concatenate `is the GOAT` to `motto`.

Reply:
`let motto = fullName;
motto += 'is the GOAT';`
Would this be better?